libraries/bot-connector/src/main/java/com/microsoft/bot/connector/authentication/OAuthConfiguration.java

```diff
@@ -10,16 +10,22 @@
 public class OAuthConfiguration {
     private String scope;
     private String authority;
+    private Boolean validateAuthority;
 
     /**
      * Construct with authority and scope.
-     * 
+     *
      * @param withAuthority The auth authority.
      * @param withScope     The auth scope.
+     * @param withValidateAuthority Whether the Authority should be validated.
      */
-    public OAuthConfiguration(String withAuthority, String withScope) {
+    public OAuthConfiguration(String withAuthority, String withScope, Boolean withValidateAuthority) {
```

Comment:
not sure why we have to add this but I guess you have good reasons 🤔. The C# class is super sparse so not much comparison.

Reply:
C# defined a class with just properties, however, Java add them also in the constructor. I think that this will cause some kind of breaking change, I think that the best approach to avoid the breacking changes is to add a new constructor with 3 parameters.

I'll apply that change